### PR TITLE
Consolidate property documentation

### LINKED ISSUE #310: Consolidate property documentation
We have two sections for documenting properties - application level deployer properties and server config properties. When properties are configurable in both styles, for the most part the content gets duplicated with only the syntax adjusted. Rather than having two sections, we should consolidate the properties part of the docs into one section to include the content and the configuration style options in the same place

### PR DESCRIPTION
**Pull request by chrisjs**
Resolves #310

### CODE REVIEW

**Comment by chrisjs** [comment]
Couple notes:

* Renamed `Application Properties` in `Getting Started` to `Application and Server Properties`

* Moved items from `Spring Cloud Deployer for Kubernetes Properties` in `Server Configuration` into the new `Application and Server Properties`

* Moved content without an existing section was added as-is. For sections that were duplicated, only the server configuration was moved into the corresponding section.

* In regards to flow, when deployer properties can override server configuration - deployer property configuration is shown first followed by the corresponding server configuration.
  
* Content itself has been left unchanged for the most part with the exception of generifying some text referring to deployer and/or server properties

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -663,13 +663,23 @@ dataflow:>task execution list
 dataflow:>task destroy --name task1
 ```
 
-== Application Configuration
+== Application and Server Properties
 
-This section covers how you can customize the deployment of your applications. You can use a number of deployer properties to influence settings for the applications that are deployed.
+This section covers how you can customize the deployment of your applications. You can use a number of properties to influence settings for the applications that are deployed. Properties can be applied on a per application basis or in the server configuration for all deployed applications.
+
+NOTE: Properties set on a per application basis will always take precedence over properties set the server configuration. This allows for the ability to override property values set at the server level.
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc]: "set the server configuration" -> "set as the server configuration"

Comment (sabbyanandan) [reply spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc]: "This allows for the ability to override property values set at the server level" -> "This allows for the ability to override global server level properties on a per-application basis."

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc [hunk 2]
```diff
@@ -852,6 +916,16 @@ deployer.<app>.kubernetes.entryPointStyle=<Entry Point Style>
 
 Replacing `<app>` with the name of your app and the desired Entry Point Style.
 
+The image pull secret can also be configured at the server level in the container env section of a deployment YAML, for example:
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc]: Maybe a copy + paste error. Perhaps this should specifically discuss the `entryPointStye`.

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc [hunk 3]
```diff
@@ -897,3 +979,13 @@ deployer.<app>.kubernetes.imagePullPolicy=Always
...
 
+An Image Pull Policy can be configured at the server level in the container env section of a deployment YAML, for example:
+
+```
+        env:
+        - name: SPRING_CLOUD_DEPLOYER_KUBERNETES_DEPLOYMENT_IMAGE_PULL_POLICY
+          value: Always
+```
+
+Replacing `Always` with the desired Image Pull Policy.
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc:990]: Should we include the link to the enum class to highlight the available options?

**Review by sabbyanandan — COMMENTED** [review]
Looks good overall. I left a few trivial change suggestions.

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc (hunk 3 above)
Comment (chrisjs) [reply spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc:990]: na i think for here the docs should be updated when the enum gets updated to include descriptions. linking the to the `KubernetesDeployerProperties` in the start of the docs i think is OK as there are many more properties than currently outlined

**Comment by chrisjs** [comment]
review comments addressed in latest push

**Review by sabbyanandan — APPROVED** [review]
Looks good.

**Comment by chrisjs** [comment]
merged in f06b4

**Pull request closed without merging**